### ISSUE TITLE
API Authentication with Terraform

### ISSUE THREAD

**Issue by AnudeepReddyJ21** [IBM-Bluemix/terraform-provider-ibm#3]
Hi ,

### Terraform Version
v0.9.11

### Provider version
tf-v0.9.3-ibm-provider-v0.2.1

### Affected Resource(s)
- ibm_infra_vlan

I exported the SL_USERNAME, SL_API_KEY variables and set 'skip_service_configuration=bluemix' 

`terraform import ibmcloud_infra_vlan.999 999`

### Expected Behavior
Terraform should have imported the state

### Actual Behavior
`import ibmcloud_infra_vlan.999 result: 999: ibmcloud_infra_vlan.999: Error communicating with the API: SoftLayer_Exception_Public: Access Denied.  (HTTP 401)`

Thanks,
Anudeep

**Comment by ashishth09** [comment]
Hi @AnudeepReddyJ21,
I am not able to reproduce the problem. Could you please re-check once if you passed the right credentials? 
Thanks

**Comment by AnudeepReddyJ21** [comment]
Hi @ashishth09 

The description of SL_USERNAME is vague. I have tried using API Username and Primary username for the account. Both of them don't seem to work. I created another user, which I guess worked. UI shows username as email id but the actual username is something else. Things are really inconsistent on the UI. 

Now I have another error. I don't see anything as VLAN ID on the UI and I cannot import using VLAN number. Where do I find the VLAN ID?

```
import ibmcloud_infra_vlan.999 result: 999: import ibmcloud_infra_vlan.999 (id: 999): Terraform detected a resource with this ID doesn't
exist. Please verify the ID is correct. You cannot import non-existent
resources using Terraform import
```

Thanks,
Anudeep
